group_doc

```diff
@@ -0,0 +1,35 @@
+#-----------------------------------------------------Part 2; Exercise #1
+#submitted by: Laura
+#updated by:
+#verified by: *needs review*
+
+SELECT product_code, product_name, list_price, discount_percent
+  FROM products;
+  
+SELECT product_code, product_name, list_price, discount_percent
+  FROM products
+ ORDER BY list_price;  
```

Comment:
I believe this line needs to be "ORDER BY list_price DESC;" in order to satisfy the "sorts the result set by list price in descending sequence" portion of the question.

Reply:
ha, yes I noticed that but figured I would leave it for someone else to update/edit since we need to do some of that. please make the change if you have not already done so. thanks!